ee-feature-pack/galleon-common/src/main/resources/feature_groups/distributable-web-local.xml

```diff
@@ -15,10 +15,7 @@
             <param name="cache-container" value="web"/>
             <param name="cache" value="sso"/>
         </feature>
-        <feature spec="subsystem.distributable-web.routing.local">
-            <param name="cache-container" value="web"/>
-            <param name="cache" value="routing"/>
-        </feature>
+        <feature spec="subsystem.distributable-web.routing.local"/>
```

Comment:
This comment is just a note for the record in case someone someday investigates this PR, or someone else reviews it now.

The two attributes that are removed above are not valid for the resource being configured. This had gone undiscovered because this feature group was never used prior to Paul's work on this issue.